validphys2/src/validphys/dataplots.py

```diff
@@ -960,12 +960,13 @@ def plot_positivity(pdfs, positivity_predictions_for_pdfs, posdataset, pos_use_k
     ax.axhline(0, color='red')
 
     posset = posdataset.load_commondata()
-    ndata  = posset.GetNData()
+    ndata  = posset.ndata
     xvals = []
 
     if pos_use_kin:
-        ax.set_xlabel('kin1')
-        xvals = [posset.GetKinematics(i, 0) for i in range(0, ndata)]
+        kin_name = "kin1"
```

Comment:
why not do `KINNAMES[0]"? Hardcoding it in one place but sharing/using KINNAMES in another place is a bit confusing.

Reply:
I agree, and hard-coded strings make it also harder to refactor. I plainly missed this.

Reply:
> why not do `KINNAMES[0]"?

Because I wasn't really looking at the strings but rather at removing the C++ methods. I've changed this together with the `_` suggestion.

Reply:
I'll stop phrasing my comments as questions ;)